vulture/core.py

```diff
@@ -282,6 +331,16 @@ def get_unused_code(self, min_confidence=0, sort_by_size=False):
         """
         Return ordered list of unused Item objects.
         """
+
+        def _has_noqa(obj):
+            code = (
+                obj.issue_code
+                if obj.issue_code in self.noqa_matches
+                else "all"
+            )
+            if code in self.noqa_matches:
+                return obj.first_lineno in self.noqa_matches[code]
```

Comment:
I think the whole function can be reduced to `return obj.first_lineno in self.noqa_matches["all"] or obj.first_lineno in self.noqa_matches[ISSUE_CODE_MAP[obj.typ]]`

Reply:
I ended up with:

```py
return obj.first_lineno in (
    self.noqa_matches[ISSUE_CODE_MAP[obj.typ]]
    or self.noqa_matches["all"]
)
```